### ISSUE TITLE
Optimize Inference Performance on CPU

### ISSUE THREAD

**Issue by carter54** [dmlc/gluon-nlp#1035]
## Description
the news in [https://github.com/dmlc/gluon-nlp/releases/tag/v0.8.1](url) shows BERT int8 quantization is presented in blog
[https://medium.com/apache-mxnet/optimization-for-bert-inference-performance-on-cpu-3bb2413d376c](url)
But the blog only shows some results of BERT quantization test, 
>  The work on low precision deployment is still ongoing and involves un-released SW, the reproduction instructions will be available later.
When will this work be released and can we apply this quantization method on GPT2? 

Thanks a lot for the great work!

**Comment by leezu** [comment]
@TaoLv

**Comment by TaoLv** [comment]
Sorry for missing the message. We're working on cleaning the code and solution. Hope we can have a PR soon. I'm not familiar with the status of GPT2 in GluonNLP. Could you please point me to the scripts and whether it can be exported as a static model?

**Comment by leezu** [comment]
Yes, recently static GPT2 model is supported: https://github.com/dmlc/gluon-nlp/pull/1010

**Comment by carter54** [comment]
Thanks for the replies. @leezu @TaoLv 
Looking forward to try int8 bert and gpt2 soon~

**Comment by TaoLv** [comment]
@carter54 FYI, here is the PR for BERT quantization: #1080

**Comment by carter54** [comment]
@TaoLv Thx for the work, can this method applied to GPT 2 model?
